Sources/FigmaExport/Resources/iOSConfig.swift

```diff
@@ -63,6 +63,8 @@ ios:
   xcassetsInMainBundle: true
   # [optional] Is Assets.xcassets located in a swift package? Default value is false.
   xcassetsInSwiftPackage: false
+  # [optional] Resource bundle name for Swift packages containing Assets.xcassets (e.g. ["PackageName_TargetName"])
+  resourceBundleNames: []
```

Comment:
```suggestion
  # [optional] When `xcassetsInSwiftPackage: true` use this property to specify a resource bundle name for Swift packages containing Assets.xcassets (e.g. ["PackageName_TargetName"]). This is necessary to avoid SwiftUI Preview crashes.
  resourceBundleNames: []
```

1. Add more information about this option.
2. Add the same information to the `CONFIG.md` file